src/Microsoft.ML.Data/DataLoadSave/DataOperationsCatalog.cs

```diff
@@ -90,5 +90,27 @@ public IDataView FilterByKeyColumnFraction(IDataView input, string columnName, d
                 throw Environment.ExceptSchemaMismatch(nameof(columnName), "filter", columnName, "KeyType", type.ToString());
             return new RangeFilter(Environment, input, columnName, lowerBound, upperBound, false);
         }
+
+        /// <summary>
+        /// Drop rows where any column in <paramref name="columns"/> contains a missing value.
+        /// </summary>
+        /// <param name="input">The input data.</param>
+        /// <param name="complement">If true, keep only rows that contain missing values, and filter the rest.</param>
+        /// <param name="columns">Name of the columns to filter on. If a row is has a missing value in any of
```

Comment:
Range filter has same option, but for some reason we decide to not expose it.

Reply:
I checked around and we will remove it here too!

---
In reply to: [254080093](https://github.com/dotnet/machinelearning/pull/2424#discussion_r254080093) [](ancestors = 254080093)